pyproject.toml

```diff
@@ -811,6 +811,7 @@ ignore = [
 
 [tool.ruff.lint.flake8-import-conventions.extend-aliases]
 voluptuous = "vol"
+"homeassistant.components.notify.PLATFORM_SCHEMA" = "NOTIFY_PLATFORM_SCHEMA"
```

Comment:
First platform - we can add more afterwards.

Reply:
In many occasions we have used `PARENT_PLATFORM_SCHEMA` in the past but we're thinking to change this to "specify" from which domain it's from?

Reply:
There is a mix of different behavior... This aims to standardize.
Standardization to `PARENT_PLATFORM_SCHEMA` is another option - I've done a single platform so we can settle on this standard before working on other platforms.

Reply:
I'd prefer the name of the platform over parent since its more descriptive

Reply:
I would agree with that 👍